src/test/java/com/epam/service/UserServiceImplTest.java

```diff
@@ -0,0 +1,20 @@
+package com.epam.service;
+
+import com.epam.entity.User;
+import org.junit.jupiter.api.Test;
+import static org.mockito.Mockito.*;
+import static org.junit.jupiter.api.Assertions.*;
+
+class UserServiceImplTest {
+    @Test
+    //simple unit test with mockito framework
+    void getUser() {
+        UserServiceImpl userService = new UserServiceImpl();
+        User user = mock(User.class);
+        when(user.getUsername()).thenReturn("John");
+        User user1 = userService.getUser(user.getUsername());
```

Comment:
Please name variable as `expected `and `actual` for increase readability. For example, use `actualUser ` or just `actual` here.
And `sout `doesn't make any sense for the test, please use asserts.